data_analysis/plots.py

```diff
@@ -695,6 +695,27 @@ def calculate_trips_per_rider(
     return daily_means
 
 
+def filter_day_type(df: pd.DataFrame, day_type: str) -> pd.DataFrame:
+    df = df.copy()
+    day_types = list(DAY_NAMES.keys())
+    if day_type not in day_types:
+        error_text = ", ".join(day_types[:-1]) + ', or ' + day_types[-1]
+        raise ValueError(f"day_type must be one of {error_text}")
+    return df.loc[df['day_type'] == day_type]
+
+
+def set_day_type_suffix(df: pd.DataFrame) -> str:
+    df = df.copy()
+    day_suffix = None
```

Comment:
Perhaps `day_suffix` could be defaulted to 'all_day_types' here so that the if condition on line 714 would be unnecessary.

Reply:
Thanks, I'll change line 709 and remove 714.